plugins/stringlib/stringlib.cpp

```diff
@@ -223,6 +229,269 @@ unsigned editDistance(unsigned leftLen, const char * left, unsigned rightLen, co
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+unsigned optimalStringAlignmentDistance(unsigned leftLen, const char * left, unsigned rightLen, const char * right)
```

Comment:
I meant prefix with the keyword "static" since it isn't exported out side of this module.